enterprise/frontend/.eslintrc.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-undef,import/no-commonjs */
+
+const TEST_FILES_NAME_PATTERN_ERROR_MESSAGE = `Please name your test setup and utils files with a ".spec.*" in the filename, or put them under "/tests", e.g. "setup.spec.ts", "MyComponent.setup.spec.ts", or "tests/setup.ts". This is to ensure they won't be imported in the SDK build.`;
```

Comment:
operations on this file.
1. Change it to js
2. Extract `baseRestrictedConfig` and use it in 2 places (see next)
3. disallow `__support__/*` on all files (1 of 2)
4. but allow them in the tests files themselves (2 of 2). This block is above storyblook block.

See how I did the same on master
https://github.com/metabase/metabase/pull/64328/files?diff=split#diff-f8f9eb3a167d9da3115be25b8b23f96b176e3bef12f033004cf45c46171197e8